src/hermes_claude_agent_sdk/parity/v4_live_scenarios.py

```
@@ -0,0 +1,318 @@
+            raise V4LiveScenarioViolation(f"{key} exceeds the 1-4 turn fixture bound")
+        bound = children.get(key, [])
+        bindings = tuple((item["call_id"], item["trial_index"], item["child_ordinal"], item["child_count"], item["path"]) for item in bound)
+        path_call_budget = tuple((path, turns if path == "positive" else 0) for path in row["mandatory_paths"])
+        surfaces = ("session", "prompt", "transcript", "stream") + _SURFACE_EXTRAS[row["mechanism_class"]]
+        if row["child_calls"]:
+            surfaces += ("child",)
+        if row["delivery_mode"] == "host_denial_local_recovery":
+            surfaces += ("delivery_boundary",)
+        scenarios.append(V4LiveScenario(
+            key, row["source_pack"], row["source_item_id"], row["predecessor_execution_id"], row["feature_id"],
+            row["mechanism_class"], "synthetic_local_fixture", f"synthetic/{key}", row["delivery_mode"], True,
```

Ruff (0.16.3)

[warning] Boolean positional value in function call

(FBT003)